index.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function(){
+
+console.log('I got here');
+
+
+
+$('.readless.hide').click(collapseText);
+$('.readmore').click(expandText);
+
+
+console.log('I got here');
+
+
+function expandText(){
+	$('#show-this-on-click').slideDown();
+	$('.readmore').hide();
+	$('.readless.hide').show();
```

Comment:
You didn't need to specifically target the `.hide` in `.readless.hide` because the JS function `.show();` would have overwritten the hide class, but it's not wrong. Just wanted to point out a different approach.